docs/en/setup/service-agent/java-agent/Plugin-list.md

```diff
@@ -0,0 +1,101 @@
+- Skywalking Agent List
+    - activemq-5.x
+    - armeria-063-084
+    - armeria-085
+    - armeria-086
+    - armeria-098
+    - avro-1.x
+    - brpc-java
+    - canal-1.x
+    - cassandra-java-driver-3.x
+    - dubbo
+    - ehcache-2.x
+    - elastic-job-2.x
+    - elastic-job-3.x
+    - elasticsearch-5.x
+    - elasticsearch-6.x
+    - feign-default-http-9.x
+    - feign-pathvar-9.x
+    - finagle
+    - graphql
+    - grpc-1.x
+    - gson-2.8.x
+    - h2-1.x
+    - hbase-1.x
+    - httpasyncclient-4.x
+    - httpclient-3.x
+    - httpclient-4.x
+    - hystrix-1.x
+    - influxdb-2.x
+    - jdk-http-plugin
+    - jdk-threading-plugin
+    - jedis-2.x
+    - jetty-client-9.0
+    - jetty-client-9.x
+    - jetty-server-9.x
+    - kafka-0.11.x/1.x/2.x
+    - kotlin-coroutine
+    - lettuce-5.x
+    - light4j
+    - mariadb-2.x
+    - memcache-2.x
+    - mongodb-2.x
+    - mongodb-3.x
+    - motan-0.x
+    - mysql-5.x
+    - mysql-6.x
+    - mysql-8.x
+    - netty-socketio
+    - nutz-http-1.x
+    - nutz-mvc-annotation-1.x
+    - okhttp-3.x
+    - play-2.x
+    - postgresql-8.x
+    - pulsar
+    - quasar
+    - rabbitmq-5.x
+    - redisson-3.x
+    - resteasy-server-3.x
+    - rocketMQ-3.x
+    - rocketMQ-4.x
+    - servicecomb-0.x
+    - servicecomb-1.x
+    - sharding-jdbc-1.5.x
+    - sharding-sphere-3.x
+    - sharding-sphere-4.0.0
+    - sharding-sphere-4.1.0
+    - sharding-sphere-4.x
+    - sharding-sphere-4.x-rc3
+    - sofarpc
+    - solrj-7.x
+    - spring-annotation
+    - spring-async-annotation-5.x
+    - spring-cloud-feign-1.x
+    - spring-cloud-feign-2.x
+    - spring-cloud-gateway-2.0.x
+    - spring-cloud-gateway-2.1.x
+    - spring-concurrent-util-4.x
+    - spring-core-patch
+    - spring-kafka-2.x
+    - spring-mvc-annotation
+    - spring-mvc-annotation-3.x
+    - spring-mvc-annotation-4.x
+    - spring-mvc-annotation-5.x
+    - spring-resttemplate-4.x
+    - spring-tx
+    - spring-webflux-5.x
+    - spymemcached-2.x
+    - struts2-2.x
+    - tomcat-7.x/8.x
+    - toolkit-counter
+    - toolkit-gauge
+    - toolkit-histogram
+    - toolkit-log4j
+    - toolkit-log4j2
+    - toolkit-logback
+    - toolkit-opentracing
+    - toolkit-tag
+    - toolkit-trace
+    - undertow-2.x-plugin
+    - vertx-core-3.x
+    - zookeeper-3.4.x   
```

Comment:
@kezhenxu94 Do you have any change to use some mechanisms to check this? Such as building a mock agent core main?

Reply:
possible

Reply:
the list is got from my shell tools, already push